File: Java/Lista2/Exerc15.java
package Lista2;

import java.util.Scanner;

public class Exerc15 {

	public static void main(String[] args) {
		// Muitos países estão passando a usar o sistema métrico.
		// Preparar um algoritmo para executar a conversão de 
		// Celsius para Fahrenheit.
		
		float C,F;
		Scanner scan  = new Scanner(System.in);
		System.out.print("Digite a temperatura em Celsius: ");
		C = Float.parseFloat(scan.next());
		
		F = (9*C+160) / 5;
		
		System.out.println("A temperatura em Fahrenheit é "+F+".");

	}

}

File: Python/Lista2/exerc15.py
# 15) Muitos países estão passando a usar o sistema métrico.
# Preparar um algoritmo para executar a conversão de Celsius para Fahrenheit.
# A fórmula de conversão é: F= (9*C+160) / 5, sendo F a temperatura em
# Fahrenheit e C a temperatura em Celsius.

C = float(input("Informe a temperatura em Celsius: "))
F = (9*C+160) / 5
print("A temperatura em Fahrenheit é ",F,".",sep="")
